components/mediation-commons/src/main/java/org/wso2/carbon/mediation/commons/rest/api/swagger/GenericApiObjectDefinition.java

```diff
@@ -109,7 +120,7 @@ public GenericApiObjectDefinition(API api, ServerConfig serverConfiguration) {
     private Map<String, Object> getInfoMap() {
         Map<String, Object> infoMap = new LinkedHashMap<>();
         infoMap.put(SwaggerConstants.DESCRIPTION, (SwaggerConstants.API_DESC_PREFIX + api.getAPIName()));
-        infoMap.put(SwaggerConstants.TITLE, api.getName());
+        infoMap.put(SwaggerConstants.TITLE, api.getAPIName());
```

Comment:
Why we are changing to API name from API name with version?

Reply:
If we call this for more than one time, this will append the version more than once. Do we have special requirement to have this name with version? version is already available in the 'Version' Swagger entry.